mailscanner/functions.php

```diff
@@ -426,12 +426,16 @@ function html_start($title, $refresh = 0, $cacheable = true, $report = false)
                 } else {
                     echo '    <tr><td colspan="3">' . __('verifyperm03') . ' ' . $incomingdir . ' ' . __('and03') . ' ' . $outgoingdir . '</td></tr>' . "\n";
                 }
-                // else use mailq which is for sendmail and exim
+                // Else use mailq which is for Sendmail and Exim
             } elseif (MAILQ) {
-                $inq = database::mysqli_result(dbquery('SELECT COUNT(*) FROM inq WHERE ' . $_SESSION['global_filter']),
-                    0);
-                $outq = database::mysqli_result(dbquery('SELECT COUNT(*) FROM outq WHERE ' . $_SESSION['global_filter']),
-                    0);
+                // If Exim, use dedicated command, else use database
+               if ($mta === 'exim') {
+                   $inq = exec('sudo /usr/sbin/exim -bpc 2>&1');
+                   $outq = exec('sudo /usr/sbin/exim -bpc -DOUTGOING 2>&1');
```

Comment:
is `OUTGOING` traversal to all exim installation or is this queue name configurable?